src/assets/queries/tags/get_descendant_folder_ids.sql

```diff
@@ -0,0 +1,7 @@
+-- Get all descendant folder IDs recursively
+with recursive descendants as (
+    select id from Folders where parentId = ?1
```

Comment:
This already exists (same for rust code equivalent) as get_child_folder_ids_recursive.sql